lunes_cms/cms/migrations/0005_alter_document_word_type.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.13 on 2022-06-29 10:01
+
+from django.db import migrations, models
+from ..models.document import reverse_changes
```

Comment:
```suggestion


# pylint: disable=unused-argument
def convert_word_type(apps, schema_editor):
    """
    Alter the word type field values from string to integer

    :param apps: The configuration of installed applications
    :type apps: ~django.apps.registry.Apps

    :param schema_editor: The database abstraction layer that creates actual SQL code
    :type schema_editor: ~django.db.backends.base.schema.BaseDatabaseSchemaEditor
    """
    document_dict = {0: "Nomen", 1: "Verb", 2: "Adjektiv"}
    document_bulk_update_list = []

    for key, value in document_dict.items():
        document = models.Document.objects.get(id=key)
        document.word_type = value
        # append the updated user object to the list
        document_bulk_update_list.append(document)
```

A few general notes about this function:
1. You need to pass the function parameters `apps` as `schema_editor` as described in [the documentation](https://docs.djangoproject.com/en/4.0/ref/migration-operations/#runpython).
2. No need do re-define the static dict `document_dict`, please import `Static.word_type_choice` instead.
3. The function itself doesn't do what it's supposed to do

Reply:
@JoeyStk This is not yet resolved. `document_dict` is still hard-coded instead of imported from the static module, and the function itself does not do what it's supposed to do.